### DISCUSSION TITLE
Using LumaFS to patch

### DISCUSSION THREAD

**Discussion by Test-Golem** [L-StarJP/Monster-Hunter-Diary-Poka-Poka-Airou-Village-DX-English-translation-3DS#1] [Ideas]
![image](https://github.com/L-StarJP/Monster-Hunter-Diary-Poka-Poka-Airou-Village-DX-English-translation-3DS/assets/173074804/4e95e4df-4a7c-4ec8-a726-09b4d6de47c4) 
from [here](https://gist.github.com/PixelSergey/5dbb4a9b90d290736353fa58e4fcbb42)

![image](https://github.com/L-StarJP/Monster-Hunter-Diary-Poka-Poka-Airou-Village-DX-English-translation-3DS/assets/173074804/e80d792d-a46e-4099-90a4-5a8f570d48e4) 
from [here](https://github.com/Omnija/Monster-Hunter-Diary-DX-3ds-Translations)

Works for me and saves the effort of decrypting + recompiling.
Note that the SD card's folder structure should look like this:
> `./luma/titles/0004000000166E00/RomFS/...`

**Comment by L-StarJP** [comment]
Thanks for the guide! Will include it in the next update.
